src/main/java/frc/robot/stateManagement/RobotStateManager.java

```diff
@@ -139,4 +124,21 @@ public DoubleSupplier getSpeakerAngle() {
     return () ->
         allianceCorrect(new Translation2d(1, Rotation2d.fromDegrees(0))).getAngle().getDegrees();
   }
+
+  public ShooterMode getShooterMode() {
+    return shooterMode;
+  }
+
+  public void setShooterMode(ShooterMode shooterMode) {
+    this.shooterMode = shooterMode;
+  }
+
+  public Translation2d getLobPosition() {
+    // TODO Auto-generated method stub
+    throw new UnsupportedOperationException("Unimplemented method 'getLobPosition'");
```

Comment:
I think this needs to be implemented before we can merge this code. I thought we talked about it being the same angle as short range mode, which would make it easy to implement, right?